hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -348,19 +350,11 @@ public static void setWALRootDir(final Configuration c, final Path root) {
   public static FileSystem getWALFileSystem(final Configuration c) throws IOException {
     Path p = getWALRootDir(c);
     FileSystem fs = p.getFileSystem(c);
-    // hadoop-core does fs caching, so need to propagate this if set
+    // hadoop-core does fs caching, so need to propogate this if set
```

Comment:
This was actually correct the first time :)

Reply:
Just reverting the previous patches @joshelser . Spelling fixes will have to come in through a different avenue.